tutorials/efas/introduction/dcs5_copper_constant_properties_electrothermal.i

```diff
@@ -0,0 +1,1444 @@
+## Parameters for the standard DCS-5 geometry and mesh, units in meters
```

Comment:
One comment here: When we have something like this running in a longer form example (more than just a syntax check), I think we'll desire a bit more information in the header than this, but I think this is OK to get into the repo and start building a tutorial on.

Reply:
I'll need an example with the amount of detail you are looking for in the examples going forward. There is a lot of variability in the examples files in the repo

Reply:
I am thinking at the level that I used in the Cincotti Examples https://github.com/idaholab/malamute/blob/devel/test/tests/example_testing/sample_one.i

Reply:
I can work with that requirement. Obviously I will not have the top level citation since this work is not replicating an example in published literature. 

Are you looking for material property citations at the top or inline as done here?

Reply:
I think inline is appropriate, and I agree about the citation. Mostly I just want more information in the header about what the input file is aiming at reproducing. Redirection to a published URL guide would also be appropriate.